src/variables/variables.js

```diff
@@ -0,0 +1,21 @@
+const ERRORS = {
+  INVALID_REQUEST: 'Invalid request',
```

Comment:
```suggestion
const ERRORS = {
  invalidRequest: 'Invalid request',
```
Keys could be in lowercase